easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.15-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,1096 @@
+easyblock = 'Bundle'
+
+name = 'R-bundle-Bioconductor'
+version = '3.15'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://bioconductor.org'
+description = """Bioconductor provides tools for the analysis and coprehension
+ of high-throughput genomic data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [('pkg-config', '0.29.2')]
```

Comment:
@branfosj Please add a clear comment why we're sticking to `pkg-config` here rather than using `pkgconf`.

We should also try and figure out why `pkgconf` isn't working as expected, since it's intended to be a drop-in replacement.
It may have something to do with exit codes of `pkgconf` vs `pkg-config` (see also https://github.com/easybuilders/easybuild-easyblocks/pull/2736)

Reply:
Ping @branfosj

Reply:
With 685cdf8 and e92925f I've added explanations into `R-bundle-Bioconductor` and `pkg-config` as to why this is being used.

I've been unable to work out how to fix the R package to build with `pkgconf`.